src/components/VideoPlayer/index.js

```diff
@@ -165,7 +165,10 @@ export default class VideoPlayer extends PureComponent {
     }
 
     if (hasAutoplay) {
-      this.video.play()
+      const promise = this.video.play()
+      if (promise !== undefined) {
```

Comment:
why is this part needed? is it undefined under certain conditions?

Reply:
not sure, it's what was recommended on the google docs.

Reply:
I'd rather play it safe-than-sorry, given this is currently plaguing the packages player

Reply:
word.  we can improve later if we find we want to